### PR TITLE
Moataz

### PR DESCRIPTION
**Pull request by Moataz199654**

### CODE REVIEW

**Comment by Moataz199654** [comment]
i removed the old folder of the CRS and removes the old folder of the SIQ
i merged them in one folder which contains the old SIQ , the old CRS and the new excel of both of them together after review.

i added the notification page flowchart, adjusted the add content flowchart and added the categories pages flowchart.

File: Design/Diagrams/DDD/Flowcharts/Add content Flowchart.pdf
Comment (AlaaOsama25) [inline Design/Diagrams/DDD/Flowcharts/Add content Flowchart.pdf:1]: Flowchart should have an start and end
user should choose from article or upload not ( article/video/record)

File: Design/Diagrams/DDD/Flowcharts/categories Flowchart.pdf
Comment (AlaaOsama25) [inline Design/Diagrams/DDD/Flowcharts/categories Flowchart.pdf:1]: Flowchart should have an start and end

File: Design/Diagrams/DDD/Flowcharts/Login and Registration Flowchart.pdf
Comment (AlaaOsama25) [inline Design/Diagrams/DDD/Flowcharts/Login and Registration Flowchart.pdf:1]: Flowchart should have an end

File: Design/Diagrams/DDD/Flowcharts/Notification page Flowchart.pdf
Comment (AlaaOsama25) [inline Design/Diagrams/DDD/Flowcharts/Notification page Flowchart.pdf:1]: Flowchart should have an start and end

**Review by AlaaOsama25 — COMMENTED** [review]
Reviewed  4 Flowchart files please find review comments

File: Design/Diagrams/DDD/Flowcharts/Login and Registration Flowchart.pdf
Comment (Moataz199654) [reply Design/Diagrams/DDD/Flowcharts/Login and Registration Flowchart.pdf:1]: Done

File: Design/Diagrams/DDD/Flowcharts/categories Flowchart.pdf
Comment (Moataz199654) [reply Design/Diagrams/DDD/Flowcharts/categories Flowchart.pdf:1]: Done

File: Design/Diagrams/DDD/Flowcharts/Add content Flowchart.pdf
Comment (Moataz199654) [reply Design/Diagrams/DDD/Flowcharts/Add content Flowchart.pdf:1]: Done

**Review by AlaaOsama25 — APPROVED** [review]

**Pull request merged**